process/accounts/accountsProcessor.go

```diff
@@ -67,13 +60,12 @@ func (ap *accountsProcessor) GetAccounts(alteredAccounts data.AlteredAccountsHan
 
 	allAlteredAccounts := alteredAccounts.GetAll()
 	for address, altered := range allAlteredAccounts {
-		userAccount, err := ap.getUserAccount(address)
-		if err != nil || check.IfNil(userAccount) {
-			log.Warn("cannot get user account", "address", address, "error", err)
+		alteredAccount := coreAlteredAccounts[address]
+		if alteredAccount == nil {
 			continue
```

Comment:
is this ok to continue here without a WARN?

Reply:
added a warn